### PR TITLE
Magic/content

### PR DESCRIPTION
**Pull request by magic-sudo**

### CODE REVIEW

File: content/contact.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: Contact
+---
+
+## Emails
+* <a href="mailto:compotesrobot@gmail.com">Team email</a>
```
Comment (pepol) [inline content/contact.md]: Please see [this link](https://guides.github.com/features/mastering-markdown/#syntax) to see how links are done.

File: content/index.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: Home
+---
+
+#Hey, what's up homies!
```
Comment (pepol) [inline content/index.md]: There should be a space between `#` and title and also an empty line after the title.

File: layouts/page.html [hunk 3]
```diff
@@ -19,11 +19,11 @@
       </div>
       <div id="menubar">
         <ul id="menu">
-          <li class="selected"><a href="index.html">Home</a></li>
-          <li><a href="examples.html">Examples</a></li>
-          <li><a href="page.html">A Page</a></li>
-          <li><a href="another_page.html">Another Page</a></li>
-          <li><a href="contact.html">Contact Us</a></li>
+          <li> <%= link_to("Home", "/") %> </li>
+          <li> <%= link_to("Aboutus", "/about") %></li>
```
Comment (pepol) [inline layouts/page.html]: About us should be written with space, shouldn't it?

File: layouts/page.html [hunk 4]
```diff
@@ -19,11 +19,11 @@
...
       <div id="menubar">
         <ul id="menu">
-          <li class="selected"><a href="index.html">Home</a></li>
-          <li><a href="examples.html">Examples</a></li>
-          <li><a href="page.html">A Page</a></li>
-          <li><a href="another_page.html">Another Page</a></li>
-          <li><a href="contact.html">Contact Us</a></li>
+          <li> <%= link_to("Home", "/") %> </li>
+          <li> <%= link_to("Aboutus", "/about") %></li>
+          <li> <%= link_to("Page", "/page") %> </li>
```
Comment (pepol) [inline layouts/page.html]: Please only put links that are **actually valid** here, not that somebody clicks a link in menu and gets 404.

**Comment by pepol** [comment]
Couple of comments @magic-sudo!

**Comment by magic-sudo** [comment]
done mein fuhrer :de:

**Comment by pepol** [comment]
@magic-sudo please keep your personal beliefs out of our collaboration platform.

File: layouts/page.html [hunk 5]
```diff
@@ -6,7 +6,7 @@
   <meta name="description" content="website description" />
   <meta name="keywords" content="website keywords, website keywords" />
   <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
-  <link rel="stylesheet" type="text/css" href="style/style.css" title="style" />
+  <link rel="stylesheet" type="text/css" href="../style/style.css" title="style" />
```
Comment (pepol) [inline layouts/page.html]: why this change?

Comment (pepol) [inline layouts/page.html]: you should consider this relative to `/` on website, so I don't think that it will be: `compotes.net/../style/style.css`, do you?

**Comment by pepol** [comment]
@magic-sudo please address my one comment and test it.
